Tests/configure_and_test_integration_instances.py

```diff
@@ -810,75 +811,6 @@ def report_tests_status(preupdate_fails, postupdate_fails, preupdate_success, po
     return testing_status
 
 
-def set_marketplace_gcp_bucket_for_build(client, prints_manager, branch_name, ci_build_number, is_nightly, is_private):
```

Comment:
how come these functions can be removed?

Reply:
```
def set_marketplace_url(servers, branch_name, ci_build_number):
    url_suffix = f'{branch_name}/{ci_build_number}'
    config_path = 'marketplace.bootstrap.bypass.url'
    config = {config_path: f'https://storage.googleapis.com/marketplace-ci-build/content/builds/{url_suffix}'}
    for server in servers:
        server.add_server_configuration(config, 'failed to configure marketplace custom url ', True)
    logging.success('Updated marketplace url and restarted servers')
    logging.info('sleeping for 60 seconds')
    sleep(60)
```
This is the one we use now it appears